### DISCUSSION TITLE
Model Engine

### DISCUSSION THREAD

**Discussion by BobtheBob123456** [Kas-tle/java2bedrock.sh#316] [Converter Help] [answered]
Question, does converter work on Model Engine Reource pack and with custom blocks?

**Comment by Kas-tle** [comment, marked as answer]
With GeyserOptionalPack it should work.

**Reply by BobtheBob123456** [reply]
Wait, what does the GeyserOptionalPack do?

**Reply by Kas-tle** [reply]
Armor stands and fixes the scaling of mini armor stands, both of which model engine needs.

**Comment by BobtheBob123456** [comment]
What about custom blocks?

**Reply by Kas-tle** [reply]
No
